### ISSUE TITLE
Frontend

### ISSUE THREAD

**Issue by muuknl** [Francismori7/ProjectManagement#12]
For the front-end we will of course be building an angularjs application. However, we have to decide whether we want to use http://material.angularjs.org, or that we want to build our own UI. I have drawn inspiration from (wouldn't open on phone since they contain big images):
 * https://dribbble.com/shots/2269224-Pipeline-UI
 * https://dribbble.com/shots/2084101-Dashboard-Overview
 * https://www.behance.net/gallery/13679177/Simplest-V2-A-flat-simple-application

So basically flat design vs. material design

**Comment by Francismori7** [comment]
I love material design however it feels too "phon-y" on a regular computer
- for me that's a no-no for me. I'm still very attracted to Bootstrap but
it is overused, that's also a no-no. I love flat UI but hate the regular
color schemes they implement, I would prefer a more colorful integration of
it while keeping it attractive for business users. In the end, we only need
it flat so no shadows, and no feeling of depth.

It's needs to be easily customizable.

Which one do you prefer?

**Comment by Francismori7** [comment]
I like the Simplest V2, very tame and very pretty.

**Comment by muuknl** [comment]
I'd prefer not going with material but with something flat. This one I also like very much: https://dribbble.com/shots/2134939-Useably-Project

I'd go for this mixed with the simplest v2.

**Comment by Francismori7** [comment]
Are they easy to combine and implement?
